packages/web-components/fast-foundation/src/radio-group/radio-group.ts

```diff
@@ -110,8 +110,11 @@ export class FASTRadioGroup extends FASTElement {
      * @remarks
      * HTML Attribute: orientation
      */
-    @attr
-    public orientation: Orientation | "horizontal" | "vertical" = Orientation.horizontal;
+    @attr({ attribute: "orientation" })
```

Comment:
we shouldn't need to enumerate the attribute here in the options as the name is no different than the property.